transit_odp/pipelines/pipelines/dataset_etl/utils/dataframes.py

```diff
@@ -184,10 +184,15 @@ def df_to_vehicle_journeys(df: pd.DataFrame) -> Iterator[VehicleJourney]:
 def df_to_serviced_organisations(
     df: pd.DataFrame, existing_serviced_orgs
 ) -> Iterator[ServicedOrganisations]:
-    unique_names = df.loc[~df["name"].isin(existing_serviced_orgs), "name"].unique()
-
-    for name in unique_names:
-        yield ServicedOrganisations(name=name)
+    unique_org_codes = df.loc[
```

Comment:
we could use drop_duplicates method from the dataframe api to drop rows with duplicate orgs.

```df.drop_duplicates(subset='serviced_org_ref', keep='first')```

and then filter orgs that are not in the db